npm-shrinkwrap.json

```diff
@@ -8564,6 +8564,12 @@
     },
     "blob": {
       "version": "0.0.4"
+    },
+    "ms": {
```

Comment:
hmmm... the shrinkwrap has conflicted everytime i rebased this. It's possible this slipped in.